### ISSUE TITLE
backspace is buggy on android

### ISSUE THREAD

**Issue by jmayergit** [40818419/react-code-input#83]
Visit the live demo using an android device.

Not sure there is a fix for this. I encountered the same problem while writing my own code. The problem lies in that backspace can either change or not change the input value.

No problems on iOS or desktop, just android.

**Comment by vispaduchuri** [comment]
yup facing the same issue, https://stackblitz.com/edit/react-invisbile-recaptcha works fine on all non-android devices. Especially once the user is at the last input tag it takes few backspaces to clear the issue and also 2 backspaces for each tag before it.
